templates/indigo/base.html

```diff
@@ -68,6 +68,29 @@
          globalSiteSpecificVars.pathToJquery = "//static.ucl.ac.uk/indigo/js/lib/jquery-2.1.1.min";
      }
     </script>
+
+    <link href="//code.jquery.com/ui/1.10.2/themes/smoothness/jquery-ui.css" rel="Stylesheet"></link>
+    <script src="//code.jquery.com/jquery-2.2.0.min.js"></script>
+    <script src="//code.jquery.com/ui/1.10.2/jquery-ui.js" ></script>
```

Comment:
Does this not have a minified version?

Reply:
No idea - I was just copying some black magic spells from a StackOverflow post...

Reply:
Seems `//code.jquery.com/ui/1.10.2/jquery-ui.min.js does exist` so you can use that.

Reply:
Though the latest is `//code.jquery.com/ui/1.12.1/jquery-ui.min.js`

Reply:
And `//code.jquery.com/jquery-2.2.4.min.js` for jQuery itself.

Reply:
And `//code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.min.css` for the CSS.

Reply:
Done.